.laminas-ci.json

```diff
@@ -9,6 +9,6 @@
         "sqlsrv"
     ],
     "ignore_php_platform_requirements": {
-        "8.1": true
+        "8.3": true
     }
```

Comment:
Remove platform ignores. There were needed for php release candidate and should not be present now.

Reply:
done